inc/functions/files.php

```diff
@@ -979,11 +979,14 @@ function rocket_clean_cache_dir() {
 	*/
 	do_action( 'before_rocket_clean_cache_dir' ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
 
-	// Delete all caching files.
-	$dirs = glob( WP_ROCKET_CACHE_PATH . '*', GLOB_NOSORT );
-	if ( $dirs ) {
-		foreach ( $dirs as $dir ) {
-			rocket_rrmdir( $dir );
+	$filesystem = rocket_direct_filesystem();
+
+	// Delete all caching files/folders.
+	foreach ( _rocket_get_cache_dirs( '.' ) as $path ) {
```

Comment:
`_rocket_get_cache_dirs()` expects an URL host as the first parameter, passing it a dot like this is strange.

How about using this instead?

```suggestion
	foreach ( _rocket_get_cache_path_iterator( _rocket_get_wp_rocket_cache_path() ) as $path ) {
```